### PR TITLE
Convert `duration` query parameter to `length` to match API

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes WordPress/openverse-api#652 by @AetherUnbound 

## Description

This is a companion PR to the API [PR#661](https://github.com/WordPress/openverse-api/pull/661). It renames the `duration` filter for audio to `length` and adds `shortest` as a value.
This PR can only be merged after we merge the companion API PR, therefore the status is currently `blocked`.

## Testing Instructions

This can only be tested with the changes in the companion API PR.
Run the API branch from #653. Set the API_URL variable for the frontend to `localhost:8000`.
Try searching for different lengths of audio using the filters. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by zackkrida** [comment]
It looks like there's a translation string issue, @obulat 

CleanShot 2022-05-17 at 22 05 31@2x

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: src/locales/en.json [hunk 1]
```diff
@@ -333,6 +333,7 @@
     },
     "durations": {
```
Comment (zackkrida) [inline src/locales/en.json:334]:
```suggestion
    "lengths": {
```

Good find, @dhruvkb

**Comment by obulat** [comment]
Thank you for catching the issue, @zackkrida , and for finding the fix, @dhruvkb  :)

**Review by dhruvkb — APPROVED** [review]

**Review by krysal — APPROVED** [review]
LGTM!

**Comment by zackkrida** [comment]
@obulat should this be merged now that WordPress/openverse-api#661 is merged and deployed?

**Comment by obulat** [comment]
We probably need to re-index, because when I checked on Friday, the API wasn't returning length.

**Comment by zackkrida** [comment]
Thanks @obulat I'll add this to our [community meeting agenda](https://docs.google.com/document/d/1AdcQVJy5B06J9N3jYCKk4GVKZZdyykjl4UP92kH_RpI/edit) for the 8th.

**Comment by krysal** [comment]
@obulat Seems like we can unblock this now that the API v2.5.5 is deployed? 😮

E.g.: https://api.openverse.engineering/v1/audio/?q=love&length=long

**Pull request merged**